src/templates/next/components/ArticleCard/ArticleCard.tsx

```diff
@@ -0,0 +1,93 @@
+import { ArticleCardProps } from "src/common/ArticleCard"
+
+import { Caption } from "../../typography/Caption"
+import { Heading } from "../../typography/Heading"
+import { Paragraph } from "../../typography/Paragraph"
+
+const ImageComponent = ({ image }: Pick<ArticleCardProps, "image">) => {
+  if (!image) return null
+  return (
+    <img
+      src={image.src}
+      alt={image.alt}
+      className="object-cover max-h-56 w-full rounded-lg"
+    />
+  )
+}
+
+const TextComponent = ({
+  lastUpdated,
+  category,
+  title,
+  description,
+  variant,
+  fileDetails,
+}: Pick<
+  ArticleCardProps,
+  | "lastUpdated"
+  | "category"
+  | "title"
+  | "description"
+  | "variant"
+  | "fileDetails"
+>) => {
+  return (
+    <div className="flex flex-col gap-3 sm:gap-8">
+      <div className={`flex flex-col gap-3`}>
+        <div className="flex flex-col sm:flex-row gap-1 sm:gap-2">
+          <p className={`${Caption[1]} text-content-medium`}>{lastUpdated}</p>
+          <p className={`${Caption[1]} hidden sm:block text-content-medium`}>
+            |
+          </p>
+          <p className={`${Caption[1]} text-content-strong`}>{category}</p>
+        </div>
+        <h4 className={`${Heading[4]} line-clamp-3 sm:line-clamp-2`}>{`${
+          variant === "file" && fileDetails
+            ? `(${fileDetails.type.toUpperCase()}) `
+            : ""
+        }${title}`}</h4>
+        <p
+          className={`${Paragraph[2]} text-content-medium line-clamp-3 sm:line-clamp-2`}
+        >
+          {description}
+        </p>
+      </div>
+      {variant === "file" && fileDetails && (
+        <div
+          className={`${Paragraph[1]} text-hyperlink underline underline-offset-2`}
+        >{`Download (${fileDetails.type.toUpperCase()}, ${
+          fileDetails.size
```

Comment:
question: wanted to check, would this be automatically populated somewhere or will the site editor be the one responsible?

Reply:
should be automatically populated when the site editor uploads a file

Reply:
Hmm okay can, I think we might have to think about how to implement this on the render engine level, not a blocker for this PR.

Some thoughts:
1. Files can only be uploaded into the `public/files` folder and should not be present in any other place.
2. This means that the article card contains the URL to the file itself, and it should contain the information inside the `_collection.json` file
3. At build time we get information about the file from the link, then compute the file size to feed into the render engine.
4. Render engine builds the page with all the props available, providing this information into the CollectionCard component.